client/components/index.js

```diff
@@ -8,4 +8,7 @@ export {default as Navbar} from './navbar'
 export {default as UserHome} from './user-home'
 export {Login, Signup} from './auth-form'
 export {default as Single_Song} from './single-song'
-export {default as Cart} from './cart/index'
+export {default as Cart} from './cart'
+export {default as Cart_Songs_View} from './cart-songs-view'
```

Comment:
why did we change these imports? My component needs the /cart/index file, that's the parent component of the entire cart